docs/docs/producers.md

```diff
@@ -220,6 +220,15 @@ Set the output queue in the `config.properties` by setting the `sqs_queue_uri` p
 
 The producer uses the [AWS SQS SDK](http://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/sqs/AmazonSQSClient.html).
 
+# Nats
+***
+To produce messages to Nats, you can have the configuration specify in `config.properties`.
+
+The configurable properties are:
+
+- `nats_url` - defaults to **localhost:4222**
+- `nats_subject` - defaults to **%{database}.%{table}**
+  - This interpolated string config controls the subject hierarchies, where `%{database}` and `%{table}` are placeholders that will be substituted at runtime
```

Comment:
we should mention that `%{type}` can also be interpolated

Reply:
comment is updated https://github.com/GlobalSport/maxwell/pull/3/files#diff-172ffaeef540749bcbd24d92f8ce364cc9e22e1d632d8a11264e8d70c0e85431R231